tests/suites/os/suite.js

```diff
@@ -97,10 +97,10 @@ module.exports = {
 		await this.context.get().os.fetch();
 
 		// If this is a flasher image, and we are using qemu, unwrap
-		if(this.suite.deviceType.data.storage.internal && (process.env.WORKER_TYPE === `qemu`)){
```

Comment:
I think we want to keep this condition, so we only try to unwrap if internal storage is supported.

Otherwise, we would try to unwrap images for boards that don't even produce flasher images.